samples/client/petstore/kotlin-nullable/src/main/kotlin/org/openapitools/client/infrastructure/ApiInfrastructureResponse.kt

```diff
@@ -33,7 +33,6 @@ class ClientError<T>(
 ) : ApiInfrastructureResponse<T>(ResponseType.ClientError)
 
 class ServerError<T>(
-    val message: String? = null,
```

Comment:
Changing this structure is a potential breaking change, as users of the generated clients may build adding these structures in the consuming code.

Reply:
I reverted this change to avoid the breaking change